dockerfiles/Dockerfile

```diff
@@ -48,16 +67,12 @@ COPY config/php/xdebug.ini /usr/local/etc/php/conf.d/xdebug.ini
 # Install Composer
 RUN curl -sS https://getcomposer.org/installer | php
 RUN mv composer.phar /usr/bin/composer
-RUN composer --version
 
 # Install Node.js
-RUN yum -y install nodejs
-RUN yum list installed nodejs
-RUN node -v
```

Comment:
execution below commands is recommended, as these will verify installed versions of needed modules
node -v
npm -v